package.json

```diff
@@ -1,13 +1,13 @@
 {
 	"name": "bitfinex-promise",
-	"version": "1.0.5",
+	"version": "1.0.6",
 	"description": "node.js wrapper for bitfinex cryptocurrency exchange using promises",
 	"main": "lib/Bitfinex.js",
     "license": "MIT",
 	"scripts": {
 		"build": "coffee --compile --output lib/ src/*.coffee",
 		"prepublish": "coffee --compile --output lib/ src/*.coffee",
-		"test": "mocha",
+		"test": "npm run build && coffee --compile --output test/ src-test/*.coffee && mocha",
```

Comment:
I'd prefer a test-compile build script.

Reply:
I don't want to sound passive-agressive, specially because I have never used coffee script before. But how did you actually run the tests before? Did you invoke both builds separately and then called mocha? What would it be the use case for running the tests without having the code compiled?

Reply:
Not passive aggressive at all and not really about the usecase.

All I'm saying is to take this part: `coffee --compile --output test/ src-test/*.coffee` and add a `"test-compile": "coffee --compile --output test/ src-test/*.coffee"` line. Then you can just do `npm run build && npm run test-compile && mocha`

Just a bit of self documenting love.

Reply:
Also, this code was written before ES6/Typescript became popular. I'd love to have it converted to TypeScript. I've got another API implementation over here: https://github.com/lookfirst/bittrex-typescript